stable/unifi/templates/ingress.yaml

```diff
@@ -2,15 +2,15 @@
 {{- $fullName := include "unifi.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
 {{- $unifiedServiceEnabled := .Values.unifiedService.enabled -}}
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
Ingress is still beta only, isn't it? https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#ingress-v1beta1-extensions

Reply:
Very nice catch, thanks!  I'll fix this.